### PR TITLE
Add PEPE token metadata

### PR DESCRIPTION
**Pull request by codingshot**
# Add PEPE token metadata

ERC-20: https://etherscan.io/address/0x6982508145454ce325ddbe47a25d4ec3d2311933
NEP-141: https://explorer.mainnet.near.org/accounts/6982508145454ce325ddbe47a25d4ec3d2311933.factory.bridge.near
Aurora ERC-20 (BlockScout): https://explorer.mainnet.aurora.dev/address/0xa30400e73c489c47ff87455c470085ad590d1a84
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0xa30400e73c489c47ff87455c470085ad590d1a84

### CODE REVIEW

File: tokens/pepe.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
...V6IiBmaWxsPSIjMDAwIi8+PHBhdGggb3BhY2l0eT0iLjYiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNOTE1LjUgMTEwMi41djFjLTEuOTE1LS4yOC0zLjU4MS4wNS01IDFoLTVjMy4wNDQtMS41MSA2LjM3OC0yLjE4IDEwLTJ6IiBmaWxsPSIjNzE3MjcxIi8+PHBhdGggb3BhY2l0eT0iLjAxIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTkxNS41IDExMDMuNWMuNzM5LS4xMyAxLjQwNi4wNCAyIC41LS45MzcuNjktMS42MDMuNTMtMi0uNXoiIGZpbGw9IiM1MzUzNTMiLz48cGF0aCBvcGFjaXR5PSIuMDEiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNODA4LjUgMTEwNS41Yy4zOTctMS4wMyAxLjA2My0xLjE5IDItLjUtLjU5NC40Ni0xLjI2MS42My0yIC41eiIgZmlsbD0iIzJCMkIyQiIvPjxwYXRoIG9wYWNpdHk9Ii4wMSIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik05MTAuNSAxMTA0LjVhOC4zOTQgOC4zOTQgMCAwIDEgNCAuNWMtMS41NjYuNzYtMi44OTkuNi00LS41ek04MTYuNSAxMTA1LjVjMS4zMzMuNjcgMS4zMzMuNjcgMCAwek05MDYuNSAxMTA2LjVjLjc1MS0xLjA3IDEuNzUxLTEuMjQgMy0uNS0uOTQ4LjQ4LTEuOTQ4LjY1LTMgLjV6TTg0OS41IDExMDYuNWMtLjM5NyAxLjAzLTEuMDYzIDEuMTktMiAuNS41OTQtLjQ2IDEuMjYxLS42MyAyLS41eiIgZmlsbD0iIzAwMCIvPjxwYXRoIG9wYWNpdHk9Ii4wMSIgZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik05MDIuNSAxMTA3LjVjLjM5Ny0xLjAzIDEuMDYzLTEuMTkgMi0uNS0uNTk0LjQ2LTEuMjYxLjYzLTIgLjV6IiBmaWxsPSIjNTM1MzUzIi8+PC9zdmc+",
```
Comment (sept-en) [inline tokens/pepe.json:6]: The icon is too big. Please make sure the icon is <10 KB.

File: tokens/pepe.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x6982508145454ce325ddbe47a25d4ec3d2311933",
+    "aurora_address": "0xa30400e73c489c47ff87455c470085ad590d1a84",
```
Comment (sept-en) [inline tokens/pepe.json:3]:
```suggestion
    "ethereum_address": "0x6982508145454ce325ddbe47a25d4ec3d2311933",
    "near_address": "6982508145454ce325ddbe47a25d4ec3d2311933.factory.bridge.near",
    "aurora_address": "0xa30400e73c489c47ff87455c470085ad590d1a84",
```

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for your PR. I added some comments below, please check.

**Review by Leena8686 — APPROVED** [review]

**Review by Leena8686 — APPROVED** [review]

**Review by Leena8686 — APPROVED** [review]

File: tokens/pepe.json (hunk 1 above)
Comment (Karkunow) [reply tokens/pepe.json:7]: @codingshot could you fix this please?

**Comment by codingshot** [comment]
https://github.com/aurora-is-near/bridge-assets/pull/257/commits/91dc2851713654730c156efa629ee76bb5ad5602

**Review by LeenaLeen8686 — COMMENTED** [review]
Approval of the

**Comment by LeenaLeen8686** [comment]
Approval granted.

**Review by LeenaLeen8686 — COMMENTED** [review]
Approval granted .

**Comment by sept-en** [comment]
@codingshot please also update the SVG file so it matches the generated code.

**Review by Leena8686 — COMMENTED** [review]
approval

**Review by Leena8686 — COMMENTED** [review]
Add PEPE token metadata is approved.

**Review by Leena8686 — COMMENTED** [review]
Add PEPE token metadata is approved.

**Comment by Karkunow** [comment]
@codingshot any updates on this one?

**Review by LeenaLeen8686 — COMMENTED** [review]
Add PEPE token Metabase and merge last 5 commits.

**Pull request closed without merging**